source/includes/api/_before_you_get_started.md

```diff
@@ -1,27 +1,60 @@
 # Before you get started
 
-This section covers a list of topics you should be familiar with before using the API. These concepts span the whole API.
+This section covers a list of topics you should be familiar with before using the API. The concepts described in this section span across multiple API endpoints and are fundamental for a better understanding of how to interact with the RW API.
 
 ## Applications
 
-TODO
+As you might have come across while reading these docs, different applications and websites rely on the RW API as the principal source for their data. While navigating through the catalog of available datasets, you will find some datasets used by the [Resource Watch website](https://resourcewatch.org/), others used by [Global Forest Watch](https://www.globalforestwatch.org/). In many cases, applications even share the same datasets!
 
+To ensure the correct separation of content across the different applications that use the RW API, you will come across a field named `application` in many of the API's resources (such as datasets, layers, widgets, and others). Using this field, the RW API allows users to namespace every single resource, so that it's associated only with the applications that use it.
 
-## Authentication
+### Officially supported applications
```

Comment:
IMO we shouldn't list the "supported applications". Instead we may want to list "currently existing" applications, and add a note on "if you want yours on this list, contact us" - but @ethanroday opinion may be more valuable here than mine

Reply:
Agree with both comments, going to update this section to reflect your feedback 👍